integration/spark2/src/main/scala/org/apache/carbondata/spark/rdd/CarbonDataRDDFactory.scala

```diff
@@ -292,6 +290,35 @@ object CarbonDataRDDFactory {
     var executorMessage: String = ""
     val isSortTable = carbonTable.getNumberOfSortColumns > 0
     val sortScope = CarbonDataProcessorUtil.getSortScope(carbonLoadModel.getSortScope)
+
+    def updateStatus(status: Array[(String, (LoadMetadataDetails, ExecutionErrors))],
```

Comment:
do not update table status file separately in separate method for size, add  size while adding loadmetadata details to table status